src/Poseidon/Janno.hs

```diff
@@ -347,9 +347,9 @@ createMinimalJanno xs = JannoRows $ map createMinimalSample xs
 createMinimalSample :: EigenstratIndEntry -> JannoRow
 createMinimalSample (EigenstratIndEntry id_ sex pop) =
     JannoRow {
-          jPoseidonID                   = id_
+          jPoseidonID                   = Bchs.unpack id_ -- TODO: this will have to change. We need to make PoseidonID itself ByteString
         , jGeneticSex                   = GeneticSex sex
-        , jGroupName                    = ListColumn [GroupName $ T.pack pop]
+        , jGroupName                    = ListColumn [GroupName . T.pack . Bchs.unpack $ pop] -- same thing, see above.
```

Comment:
I guess that's related to https://github.com/poseidon-framework/poseidon-hs/issues/331

Reply:
Yes.